src/Query.php

```diff
@@ -139,4 +193,11 @@ public function api()
     {
         return resolve(Api::class);
     }
+
+    protected function processResults($results): Collection
```

Comment:
This name feels a bit vague. What about something like `hydrateDocuments`?

Reply:
I've made this change